component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -309,12 +309,40 @@ private static String getEncodingMethodFromRegistry(String tenantDomain, Authent
 		return encodingMethod;
 	}
 
+	/**
+	 * Get stored time step size.
+	 *
+	 * @param tenantDomain Tenant domain name.
+	 * @return Time step size.
+	 * @throws AuthenticationFailedException On Error while getting value for time step size from registry.
+	 */
+	public static Long getTimeStepSize(String tenantDomain) throws AuthenticationFailedException {
```

Comment:
Not formatted

Reply:
addressed with https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/77/commits/9441e9b18f9e89e6d4184cf0beef797878c68e7d